libs/spark/src/CardContent.tsx

```diff
@@ -0,0 +1,5 @@
+export const MuiCardContentStyleOverrides = {
+  root: {
+    padding: '24px 24px 0',
```

Comment:
Can you check if this works & include if you think it's a good idea. Finding the right paradigm for padding when the children can vary is always hard.
```suggestion
    padding: '24px 24px 0',
    '&:last-child': {
      paddingBottom: 24;
    }
```

Reply:
The last-child rule is already present from the Mui default styles, so this combination of rules already applies to MuiCardContent.

Reply:
Ah okay, great